cli/tools/test/fmt.rs

```diff
@@ -318,7 +318,8 @@ pub const OP_DETAILS: phf::Map<&'static str, [&'static str; 2]> = phf_map! {
   "op_fsync_async" => ["flush pending data operations for a file to disk", "awaiting the result of a `file.fsync` call"],
   "op_ftruncate_async" => ["truncate a file", "awaiting the result of a `Deno.ftruncate` call"],
   "op_funlock_async" => ["unlock a file", "awaiting the result of a `Deno.funlock` call"],
-  "op_futime_async" => ["change file timestamps", "awaiting the result of a `Deno.futime` call"],
+  "op_fs_utime_async" => ["change file timestamps", "awaiting the result of a `Deno.futime` call"],
+  "op_fs_file_utime_async" => ["change file timestamps", "awaiting the result of a `Deno.FsFile#utime` call"],
```

Comment:
Is `Deno.FsFile#utime` correct? Or is `Deno.FsFile.utime` or something else better?